### PR TITLE
improved definition for mDCv

### PR DESCRIPTION
**Pull request by digitaltvguy**
and also modified "should" to "may" as per Simon's suggestion

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3982,15 +3999,28 @@ <h2><span class="chunk">cLLi</span> Content Light Level Information</h2>
...
-             average light level (in cd/m<sup>2</sup>, also known as nits) of the entire playback sequence. MaxFALL is
-             calculated by first averaging the decoded luminance values of all the pixels in each frame,
-             and then using the value for the frame with the highest value.</p>
+          <p>The <span class="chunk">cLLi</span> chunk adds static metadata which provides an opportunity
+          to optimize tone mapping of the associated content to a specific target display. This is 
+          accomplished by tailoring the tone mapping of the content itself to the specific peak brightness 
+          capabilities of the target display to prevent clipping. The method of tone-mapping optimization 
+          is currently subjective.</p>
+             
+          <p>MaxFALL (Maximum Frame Average Light Level) uses a static metadataa value to indicate the
```
Comment (fintelia) [inline index.html:4008]: nit: metadataa -> metadata

Comment (svgeesus) [inline index.html:4008]:
```suggestion
          <p>MaxFALL (Maximum Frame Average Light Level) uses a static metadata value to indicate the 
```

File: index.html [hunk 2]
```diff
@@ -3887,12 +3899,13 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
...
 
-          <p>Below are mDCv examples for [[SMPTE-ST-2086]] with <a>HDR</a> content that may be native or
-            containerized in <a>HDR</a> content (as described in the [[MovieLabs-Recommended-Best-Practice-for-SDR-to-HDR-Conversion]]).</p>
+           <p>Below are mDCv examples for [[SMPTE-ST-2086]] with <a>SDR</a> [[ITU-R-BT.709]] content that may be native or 
+            containerized in <a>HDR</a> [[ITU-R-BT.2100]]content (as described in the [[MovieLabs-Recommended-Best-Practice-for-SDR-to-HDR-Conversion]]).</p>
 
 
           <aside class="example">
-            Example <span class="chunk">mDCv</span> chunk mastering display colour primaries for <a>SDR</a>:
+            Example <span class="chunk">mDCv</span> chunk mastering display colour primaries for SDR[[ITU-R-BT.709]]:
```
Comment (svgeesus) [inline index.html:3907]:
```suggestion
            Example <span class="chunk">mDCv</span> chunk mastering display colour primaries for SDR [[ITU-R-BT.709]]:
```

File: index.html [hunk 3]
```diff
@@ -3746,20 +3746,28 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
...
+          allows a target (consumer) display to potentially optimize it's tone mapping decisions 
+          on a comparison of its inherent capabilities versus the original mastering displays capabilites.</p>
+          
+          <p>mDCv is typically used with the <a></a>PQ</a>[[ITU-R-BT.2100] transfer function
+          and is commonly then called HDR10 (PQ with ST 2086). The mDCv chunk may be 
+          included with <a>PQ</a> [[ITU-R-BT.2100]], <a>SDR</a> [[ITU-R-BT.709]] and <a>HLG</a> [[ITU-R-BT.2100]] 
+          and other image formats (even though mDCv use in some of them may be less common). 
+          
+          Color Primaries and White Point characteristics can be derived from cICP chunk formats. 
+          Specific examples of its most common use-cases for images using both HDR[[ITU-R-BT.2100]]
```
Comment (svgeesus) [inline index.html:3762]:
```suggestion
          Specific examples of its most common use-cases for images using both HDR [[ITU-R-BT.2100]] 
```

File: index.html [hunk 4]
```diff
@@ -3746,20 +3746,28 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
...
+          on a comparison of its inherent capabilities versus the original mastering displays capabilites.</p>
+          
+          <p>mDCv is typically used with the <a></a>PQ</a>[[ITU-R-BT.2100] transfer function
+          and is commonly then called HDR10 (PQ with ST 2086). The mDCv chunk may be 
+          included with <a>PQ</a> [[ITU-R-BT.2100]], <a>SDR</a> [[ITU-R-BT.709]] and <a>HLG</a> [[ITU-R-BT.2100]] 
+          and other image formats (even though mDCv use in some of them may be less common). 
+          
+          Color Primaries and White Point characteristics can be derived from cICP chunk formats. 
+          Specific examples of its most common use-cases for images using both HDR[[ITU-R-BT.2100]] 
+          and SDR[[ITU-R-BT.709]] are available in [[ITU-T-Series-H-Supplement-19]].</p>
```
Comment (svgeesus) [inline index.html:3763]:
```suggestion
          and SDR [[ITU-R-BT.709]] are available in [[ITU-T-Series-H-Supplement-19]].</p>
```

File: index.html [hunk 5]
```diff
@@ -3954,8 +3970,9 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
           </aside>
 
           <aside class="example">
-            Example <span class="chunk">mDCv</span> chunk mastering display maximum
-            luminance using "single-master" live television production techniques at 203 cd/m<sup>2</sup>
+
+            Example <span class="chunk">mDCv</span> chunk mastering display maximum 
+          luminance using "single-master" <a>SDR</a> [[ITU-R-BT.709]]shading techniques at 203 cd/m<sup>2</sup>
```
Comment (svgeesus) [inline index.html:3975]:
```suggestion
          luminance using "single-master" <a>SDR</a> [[ITU-R-BT.709]] shading techniques at 203 cd/m<sup>2</sup> 
```

File: index.html [hunk 6]
```diff
@@ -7471,8 +7502,9 @@ <h3 id="changes-20031110">Changes since the <a href="https://www.w3.org/TR/2003/
       </li>
 
       <li>Added the <a class="chunk" href="#cICP-chunk">cICP</a> chunk, Coding-independent code points for video signal type
-      identification, to contain image format metadata defined in [[ITU-T-H.273]] which enables PNG to contain High Dynamic Range
+      identification, to contain image format metadata defined in [[ITU-T-H.273]] which enables PNG to contain [[ITU-R-BT.2100]]High Dynamic Range
```
Comment (svgeesus) [inline index.html:7505]:
```suggestion
      identification, to contain image format metadata defined in [[ITU-T-H.273]] which enables PNG to contain [[ITU-R-BT.2100]] High Dynamic Range
```

File: index.html [hunk 7]
```diff
@@ -7442,14 +7473,14 @@ <h3 id="changes-20221025">Changes since the <a href="https://www.w3.org/TR/2022/
       <li>Redrew chunk-ordering lattice diagrams to be clearer and more consistent.</li>
       <li>Added a new chunk, <a href="#cLLi-chunk" class="chunk">cLLi</a>,
       to describe the Maximum Single-Pixel and Frame-Average Luminance Levels
-      for both static and animated <a>HDR</a> PNG images.
+      for both static and animated <a>HDR</a> [[ITU-R-BT.2100]]PNG images.
```
Comment (svgeesus) [inline index.html:7476]:
```suggestion
      for both static and animated <a>HDR</a> [[ITU-R-BT.2100]] PNG images.
```

**Review by svgeesus — APPROVED** [review]

**Pull request merged**